extensions/wikia/AdEngine/AdEngine2ContextService.class.php

```diff
@@ -66,7 +66,8 @@ public function getContext( Title $title, $skinName ) {
 					'pageName' => $title->getPrefixedDBKey(),
 					'pageType' => $wikiaPageType->getPageType(),
 					'sevenOneMediaSub2Site' => $wg->AdDriverSevenOneMediaOverrideSub2Site,
-					'skin' => $skinName,
+					// TODO: determine mercury skin in the right way
+					'skin' => $skinName === 'oasis' ? 'oasis' : 'mercury',
```

Comment:
ping @nandy-andy 
2 times duplicate this logic in 2 different JS modules VS broken ads on `Wikiamobile`
do you have another idea?